components/post_view/message_attachments/message_attachment/message_attachment.test.jsx

```diff
@@ -169,4 +178,18 @@ describe('components/post_view/MessageAttachment', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should match snapshot when the footer is truncated', () => {
```

Comment:
I miss a test case where there is no footer at all.

Reply:
Thanks @jespino, I added a snapshot test case for an attachment without a footer.